### PR TITLE
Update support and advise content

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Update support and advise content, and the "is there something wrong with this page?" section, in line with the prototype.

### Guidance to review

Find a course in the review app and scroll down to the "Support and advice" section.

### Screenshot

image

### Trello card

https://trello.com/c/SCOj6cWU/847-updating-content-and-links-to-git-from-find

### Checklist

- [x] Rebased `main`
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/views/courses/_advice.html.erb [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
+    get school experience, and find events where you can speak to training providers and teachers.</p>
+  <p class="govuk-body">Call Get Into Teaching free of charge on <%= t("get_into_teaching.tel") %>, or
     <%= govuk_link_to 'chat online', t('get_into_teaching.url_online_chat') %>
     <%= t('get_into_teaching.opening_times') %> (except public holidays).</p>
 
-  <p class="govuk-body">Find out about the different
-  <%= govuk_link_to('ways to train', t('get_into_teaching.url_ways_to_train')) %>
-  if you do not have a degree, or are already an experienced teacher.</p>
-
+  <h3 class="govuk-heading-m">Is there something wrong with this page?</h3>
```
Comment (avinhurry) [inline app/views/courses/_advice.html.erb:10]: Not part of this card, but i've updated this section in line with the prototype as I noticed they were different @Kateharries @simonwhatley is that ok?

**Comment by avinhurry** [comment]
[Updating content and links to GIT from Find](https://trello.com/c/SCOj6cWU/847-updating-content-and-links-to-git-from-find)

File: config/locales/en.yml [hunk 2]
```diff
@@ -84,6 +84,7 @@ en:
     url_bursaries_and_scholarships: https://getintoteaching.education.gov.uk/funding-your-training#bursaries-and-scholarships
     url_improve_your_subject_knowledge: https://getintoteaching.education.gov.uk/improve-your-subject-knowledge
     url_teacher_training_events: https://getintoteaching.education.gov.uk/events?utm_source=find-postgraduate-teacher-training.service.gov.uk&utm_medium=referral&utm_campaign=find_banner
+    url_teacher_training_advisor: https://getintoteaching.education.gov.uk/teacher-training-advisers
```
Comment (defong) [inline config/locales/en.yml:87]:
```suggestion
    url_teacher_training_adviser: https://getintoteaching.education.gov.uk/teacher-training-advisers
```

Comment (avinhurry) [reply config/locales/en.yml]: https://github.com/DFE-Digital/find-teacher-training/pull/1625/commits/b0b08a818d1f6771e58309e9dcda94f045ca6132

File: app/views/courses/_advice.html.erb (hunk 1 above)
Comment (Kateharries) [reply app/views/courses/_advice.html.erb:10]: Hello. Yes, that's fine - thanks @avinhurry :)

**Review by Kateharries — APPROVED** [review]
Looks good.

**Review by defong — APPROVED** [review]

**Pull request merged**